airbyte-integrations/connectors/source-bing-ads/pyproject.toml

```diff
@@ -29,7 +29,7 @@ source-bing-ads = "source_bing_ads.run:run"
 freezegun = "^1.4.0"
 pytest-mock = "^3.6.1"
 pytest = "^8.0.0"
-requests-mock = "^1.9.3"
+requests-mock = "^1.12.1"
```

Comment:
Not needed but I had requests-mock issues and I know previous versions were having concurrency issues